src/zcl/zclFrame.ts

```diff
@@ -82,6 +82,7 @@ class ZclFrame {
 
     private writeHeader(buffalo: BuffaloZcl): void {
         const frameControl = (
+            this.Header.frameControl.reservedBits | 
```

Comment:
What is the purpose of reserved bits.

Reply:
It's part of the control frame according to the ZigBee standart: 5-7 bits. Masked them by 0xE0. I know that it's suggested to be 0, but Livolo uses them. 
After these changes I'm going to update converters for their new sockets and switches. 
https://github.com/Koenkk/zigbee2mqtt/issues/3560

Need only understand how to setup modelID manually in configuration file, like we do for 'friendly_name', so if you can help here it would be great.